Correct the schema identifier.json

christoph-maurer opened the pull request:
The tests of uuidWithHyphen.json and uuidWithoutHyphen.json against
identifier.json are both valid. However,
smallGeneralComponentWithoutHyphen.json is invalid against dbe.json:
```
./examples/dbe/smallGeneralComponentWithoutHyphen.json invalid
[
  {
    instancePath: '/components/0/id',
    schemaPath: '#/$defs/central/format',
    keyword: 'format',
    params: { format: 'uuid' },
    message: 'must match format "uuid"'
  },
  {
    instancePath: '/components/0/id',
    schemaPath: '#/$defs/decentral/type',
    keyword: 'type',
    params: { type: 'object' },
    message: 'must be object'
  },
  {
    instancePath: '/components/0/id',
    schemaPath: '#/oneOf',
    keyword: 'oneOf',
    params: { passingSchemas: null },
    message: 'must match exactly one schema in oneOf'
  }
]
```
@simon-wacker Do you know why?

simon-wacker commented:
As the first error message says, it's not a valid UUID. We use [`ajv-formats`](https://github.com/ajv-validator/ajv-formats#formats) to test for validity of UUIDs, which uses the standard [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122#page-4), and in this hyphens are not allowed.

christoph-maurer commented:
I have corrected identifier.json. The definition of `decentral` was lacking `additionalProperties`: false
and `required`: [`value`,`issuedBy`]. I added some tests and corrected some examples. 

I have renamed `CRI` to `ColorRenderingIndex`.
